src/index.js

```diff
@@ -0,0 +1,162 @@
+const sample = {
```

Comment:
You could have put this object in a different .js file and still used it in index.js, in order to have a cleaner file structure.